### ISSUE TITLE
Incorrect start/end connector length for last part record in "braceleft" glyph assembly

### ISSUE THREAD

**Issue by kevinbarabash** [stipub/stixfonts#207]
Here are the values for "braceleft" glyph assembly:
```
        <GlyphAssembly>
          <ItalicsCorrection>
            <Value value="0"/>
          </ItalicsCorrection>
          <!-- PartCount=5 -->
          <PartRecords index="0">
            <glyph value="uni23A9"/>
            <StartConnectorLength value="0"/>
            <EndConnectorLength value="500"/>
            <FullAdvance value="1275"/>
            <PartFlags value="0"/>
          </PartRecords>
          <PartRecords index="1">
            <glyph value="uni23AA"/>
            <StartConnectorLength value="1000"/>
            <EndConnectorLength value="1000"/>
            <FullAdvance value="1001"/>
            <PartFlags value="1"/>
          </PartRecords>
          <PartRecords index="2">
            <glyph value="uni23A8"/>
            <StartConnectorLength value="500"/>
            <EndConnectorLength value="500"/>
            <FullAdvance value="1947"/>
            <PartFlags value="0"/>
          </PartRecords>
          <PartRecords index="3">
            <glyph value="uni23AA"/>
            <StartConnectorLength value="1000"/>
            <EndConnectorLength value="1000"/>
            <FullAdvance value="1001"/>
            <PartFlags value="1"/>
          </PartRecords>
          <PartRecords index="4">
            <glyph value="uni23A7"/>
            <StartConnectorLength value="600"/> // should be 500
            <EndConnectorLength value="500"/> // should be 0
            <FullAdvance value="1275"/>
            <PartFlags value="0"/>
          </PartRecords>
```
Some of the values for the last part record are incorrect.  The start connector length should be `500` and the end connector length should be `0`.  I looked at the "braceright" glyph assembly and these are the values it uses for the last part record.

**Comment by kevinbarabash** [comment]
@tiroj would you accept a PR for this?  I should be able to fix this using ttx.

**Comment by tiroj** [comment]
I’m not sure if I am supposed to merge PRs in the public repo. STIX Two development work is still being done in a private, dev repo, in milestone waves, and then STI Pub migrates it to the public repo. It’s what they’re comfortable with at the moment.

Source-wise, the version of the MATH table that would need to be updated is the one in the `STIXTwoMath-Regular.input.ttf` file in the `source` folder. The build process grabs the MATH table from that and copies it into the final fonts (along with the format 14 cmap and a few other things). Then we would need to rebuild the fonts using the updated input.

**Comment by kevinbarabash** [comment]
I see.  No worries then.  I'll fix the issue in my local copy of STIXTwoMath_regular.otf for now.

**Comment by tiroj** [comment]
**b171.** Hap opportunity, so fixed for v2.13 b171.
